README.md

```diff
@@ -231,12 +231,12 @@ Examples to start the fictitious file named `foo_measurement.py`:
 
 ### Create Executable for Python Scripts
 
-To create an executable from a measurement, measurement authors can use the [pyinstaller](https://www.pyinstaller.org/) tooling. During the executable creation, the user can also embed the User Interface file using the `--add-data "<path_of_the_UI_File>;."`.
+To create an executable from a measurement, measurement authors can use the [pyinstaller](https://www.pyinstaller.org/) tooling. If you are using a virtual environment, ensure that pyinstaller is installed to your venv. During the executable creation, the user can also embed the User Interface file using the `--add-data "<path_of_the_UI_File>;."`. The `.serviceconfig` file will need to be included using the `--add-data` option as well.
```

Comment:
Combine the sentences about `--add-data`.

Is it necessary for this sentence to provide an example of the option argument? One is provided below.